src/sentry/conf/server.py

```diff
@@ -1043,6 +1043,8 @@ def SOCIAL_AUTH_DEFAULT_USERNAME():
     "organizations:transaction-name-normalize": False,
     # Try to derive normalization rules by clustering transaction names.
     "organizations:transaction-name-clusterer": False,
+    # Sanitize transaction names in the ingestion pipeline
+    "organizations:transaction_name_sanitization": False,
```

Comment:
All the other feature flags use dashes

```suggestion
    "organizations:transaction-name-sanitization": False,
```